sdk/storage/Azure.Storage.DataMovement/KnownIssues.md

```diff
@@ -26,4 +26,40 @@ The Azure Storage REST service has various limitations as to the size of request
 
 ### Resume Behavior when Client Options are set
 
-When calling `*StorageResourceProvider.FromClient` with a Storage Client (e.g. `BlobBaseClient`, `ShareFileClient`) initialized with `*ClientOptions` (e.g `BlobClientOptions`, `ShareClientOptions`). It is NOT guaranteed that properties set within the `*ClientOptions` will be respected when resuming a transfer. 
\ No newline at end of file
+When calling `*StorageResourceProvider.FromClient` with a Storage Client (e.g. `BlobBaseClient`, `ShareFileClient`) initialized with `*ClientOptions` (e.g `BlobClientOptions`, `ShareClientOptions`). It is NOT guaranteed that properties set within the `*ClientOptions` will be respected when resuming a transfer.
+
+### Encryption Scope Support
+
+There is currently no support to specify an encryption scope for operations such as uploads, downloads, or copies.
+
+### Recursive Directory Support
+
+There is currently no support to handle operations that involve processing an entire directory, including all its subdirectories and files, automatically in a single command.
+
+### Blob to File with OAuth/Entra ID
+
+There is currently no support for transferring data from an Azure Blob Storage to a file system using authentication based on Microsoft Entra ID.
+
+### Resume Transfers started from any Beta version
+
+Transfers initiated with any beta version cannot be resumed due to a schema version change from string to int, which may lead to unexpected errors.
+
+### Tag Support
+
+There is currently no support for managing or utilizing tags. This means users cannot add, retrieve, or manage blob tags during a transfer.
+
+### Public URL Copying
+
+There is currently no support for copying blobs via Public URLs. However, the library does support copying Azure Storage URLs, which can be used for moving data within the Azure Storage ecosystem.
+
+### Copy with a AzureSasCredential
+
+There is currently no support for service-to-service copy given a client that was initialized with a AzureSasCredential from BlobStorageResourceProvider.FromClient and/or ShareStorageResourceProvider.FromClient.
```

Comment:
I think here we can add 2 mitigations for this.

Like if you would like to use a AzureSasCredential object to authenticate, we recommend using BlobsStorageResourceProvider(AzureSasCredential) / ShareFilesStorageResourceProvider(AzureSasCredential) and creating your respective StorageResource from there.

You can also initialize a client with a Uri that has a SAS included in the Uri if you wanted to still use BlobStorageResourceProvider.FromClient and/or ShareStorageResourceProvider.FromClient.

Something like that